cmd/infracost/generate.go

```diff
@@ -0,0 +1,95 @@
+package main
+
+import (
+	"io"
+	"os"
+
+	"github.com/spf13/cobra"
+
+	"github.com/infracost/infracost/internal/config/template"
+	"github.com/infracost/infracost/internal/ui"
+)
+
+type generateConfigCommand struct {
+	projectPath  string
+	templatePath string
+	outFile      string
+}
+
+func newGenerateConfigCommand() *cobra.Command {
+	var gen generateConfigCommand
+
+	cmd := &cobra.Command{
+		Use:   "config",
+		Short: "Generate Infracost config file from a template file",
+		Long:  "Generate Infracost config file from a template file",
+		Example: `
+      infracost generate config --project-path . --template-path infracost.yml.tmpl
+      `,
+		ValidArgs: []string{"--", "-"},
+		RunE:      gen.run,
+	}
+
+	cmd.Flags().StringVar(&gen.projectPath, "project-path", ".", "Path to the Terraform project you want to run the template file on")
+	cmd.Flags().StringVar(&gen.templatePath, "template-path", "", "Path to the Infracost template file which will generate the yml output")
```

Comment:
@hugorut maybe `Path to the Infracost template file that will generate the config-file yaml output` ?